src/test/java/map/BattleMapTest.java

```diff
@@ -201,4 +201,52 @@ void setCellStateChangingFromHealthyToSunkOnlyForLone() {
 
     assertEquals(sunkCellState.getValue(), battleMap.getAt(point), "No sunk position found at " + point);
   }
+
+  @Test
+  @DisplayName("A empty cell state has to be free")
+  void checkTheCellStateItIsBusyOrNotWhenTheCellValueIs0() {
```

Comment:
Please switch to snake case for all the test names and try to be more specific in test names. E.g. this test checks whether the cell is free when its value is `0`. We could name it `a_empty_cell_state_has_to_be_free` instead of `checkTheCellStateItIsBusyOrNotWhenTheCellValueIs0` since this is the case for the test.